lld/test/ELF/lto/devirt_validate_vtable_typeinfos.ll

```diff
@@ -125,17 +125,17 @@
 
 ;; Index based WPD
 ; RUN: ld.lld %t1.o %t2_nortti.o -o %t8_index -save-temps --lto-whole-program-visibility --lto-validate-all-vtables-have-type-infos \
-; RUN:   --lto-known-safe-vtables=_ZTV6Native -mllvm -pass-remarks=. 2>&1 | FileCheck %s --check-prefix=REMARK
+; RUN:   --lto-known-safe-vtables=_ZTV6N* -mllvm -pass-remarks=. 2>&1 | FileCheck %s --check-prefix=REMARK
```

Comment:
Consider quotes `'_ZTV6N*'`, otherwise I suspect this may fail in certain cases using an external shell when `nullglob` is enabled.